pkgs/applications/misc/wallgen/default.nix

```diff
@@ -0,0 +1,34 @@
+with import <nixpkgs> { };
+with pkgs.python3Packages;
+
+buildPythonApplication rec {
+
+  pname = "wallgen";
+  version = "cf9cfbd";
```

Comment:
See https://nixos.org/manual/nixpkgs/stable/#sec-package-naming: 
`If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".`